generation/commands/main.js

```diff
@@ -12,12 +12,18 @@ const { isEmpty } = require("lodash")
 const apiInfo = require("../schema/generated-api-schema.json")
 const descriptionOverrides = require("../schema/description-overrides.json")
 const localFlags = require("../schema/local-flags.json")
+const localArgs = require("../schema/local-args.json")
 const ApiCommand = require("./api-command.js")
 const mapChars = require("./character-mapping")
 
 // Mapping of the names of flags to the characters to use for those flags
 const charMap = mapChars(apiInfo)
 
+let commandsWithTail = new Map()
+commandsWithTail.set("logs", ["filter", "list"])
```

Comment:
Instead of hardcoding this into the generation script, could we add this property to the JSON (manually, for now)? This would make it simple to add the tail feature to a new command by just adding a `tail: true` property to that command in the JSON.

Reply:
I agree that if would be nice to have the ability of tail in the json, I worry it will cause more issues in the future. Tail is a unique to cli and it won't be found in generated JSON unless manually put in. Which means until we have an oas that is maintained it might be forgotten. The way it is set up now, you have give any command a tail by adding the topic and command to the map